### ISSUE TITLE
Pavement Markings Work Orders AGOL feature layer not populating 

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#11760]
**Describe the bug**
The pavement markings work order AGOL feature layer is not updating and the segments are not showing. 

**URL**
https://austin.maps.arcgis.com/home/item.html?id=a9f5be763a67442a98f684935d15729b

**Comment by andrewshensky** [comment]
@johnclary This appears to be the same issue as before related to the ETL. Can you take a look at this one?

Similar previous issues:
https://github.com/cityofaustin/atd-data-tech/issues/11514
https://github.com/cityofaustin/atd-data-tech/issues/10823

**Comment by johnclary** [comment]
@andrewshensky yes—will take a look tomorrow. I will also take another look at the ETL again to see if i can figure out where the breakdown is happening.

**Comment by johnclary** [comment]
i'm taking a look now—sorry for the delay!

**Comment by johnclary** [comment]
@ChristinaTremel this is fixed. As in past instances I took a look at the ETL logs (in Airflow) and cannot identify what exactly is going on here. 

The solution is always to run the ETL locally with no date argument:

```shell
$ docker run -it --rm --env-file env_file atddocker/atd-knack-services:production services/agol_build_markings_segment_geometries.py -l markings_work_orders

$ docker run -it --rm --env-file env_file atddocker/atd-knack-services:production services/agol_build_markings_segment_geometries.py -l markings_jobs
```

It's not possible to just re-trigger the DAG in Airflow, because we want the ETL to run with no `--date` arg.
